src/d3a/d3a_core/simulation.py

```diff
@@ -279,9 +280,10 @@ def _execute_simulation(self, slot_resume, tick_resume, console=None):
         self.simulation_config.external_redis_communicator.start_communication()
         self._update_and_send_results()
         for slot_no in range(slot_resume, slot_count):
+
             self._update_progress_info(slot_no, slot_count)
 
-            log.info(
+            log.warning(
```

Comment:
Probably a remnant of testing. Pleas revert to INFO. Thanks

Reply:
Will do, thanks!